csharp/test/Microsoft.ML.OnnxRuntime.EndToEndTests/runtest.sh

```diff
@@ -3,7 +3,6 @@
 # Licensed under the MIT License.
 
 LocalNuGetRepo=$1
-SourceRoot=$2
```

Comment:
why not keep this parameter and pass the correct value? seems better to not depend on azure devops variable names like BUILD_SOURCESDIRECTORY outside of build definitions.

Reply:
could add an additional parameter for the binaries directory

Reply:
Yes, then I need to add another parameter, which requires modify more files(all the callers of this script). That's why I choose this.

Reply:
Indeed I would prefer to pass less parameters when possible. The more we have, the more likely we could make some errors during the passing, for example, missing one parameter or the order is wrong.

Reply:
that's what error handling is for :)

Reply:
Oh, I forgot to mention, sometimes the file is called from docker, sometimes it runs directly in the host environment.